hook-api/src/config.rs

```diff
@@ -16,6 +16,9 @@ pub struct Config {
 
     #[envconfig(default = "100")]
     pub max_pg_connections: u32,
+
+    #[envconfig(default = "5000000")]
```

Comment:
I tried, but this doesn't support anything but a plain number (i.e. no `5_000_000`)